supervisor/addons/utils.py

```diff
@@ -44,13 +44,15 @@ def rating_security(addon: AddonModel) -> int:
         any(
             privilege in addon.privileged
             for privilege in (
+                Capabilities.BPF,
+                Capabilities.DAC_READ_SEARCH,
                 Capabilities.NET_ADMIN,
                 Capabilities.NET_RAW,
+                Capabilities.PERFMON,
                 Capabilities.SYS_ADMIN,
-                Capabilities.SYS_RAWIO,
-                Capabilities.SYS_PTRACE,
                 Capabilities.SYS_MODULE,
-                Capabilities.DAC_READ_SEARCH,
+                Capabilities.SYS_PTRACE,
+                Capabilities.SYS_RAWIO,
             )
```

Comment:
I just wonder why this list is missing SYS_TIME and SYS_RESOURCE, for example.

Reply:
Oh, now I understand.

Question: should we really decrease 1 point for BPF and PERFMON? Maybe not for PERFMON?

Reply:
I'd say so yes. There is a reason for the `perf_event_paranoid` config, perf monitor has an attack surface... See e.g. https://patchwork.kernel.org/project/linux-hardening/patch/1469630746-32279-1-git-send-email-jeffv@google.com/.

Reply:
Got it. Thanks.